MinimedKit/Messages/ReadSettingsCarelinkMessageBody.swift

```diff
@@ -52,11 +52,20 @@ public class ReadSettingsCarelinkMessageBody: CarelinkLongMessageBody {
         guard rxData.count == type(of: self).length else {
             return nil
         }
-
-        let maxBolusTicks: UInt8 = rxData[7]
+        
+        let newer = rxData[7] == 25 // x23
```

Comment:
Interesting. Does this constant appear in other messages with shifts, like read reservoir volume?